apps/server/src/modules/deletion/entity/deletion-request.entity.ts

```diff
@@ -15,19 +14,19 @@ export interface DeletionRequestEntityProps {
 }
 
 @Entity({ tableName: 'deletionrequests' })
-@Index({ properties: ['targetRefId', 'targetRefDomain'] })
+@Unique({ properties: ['targetRefId', 'targetRefDomain'] })
 export class DeletionRequestEntity extends BaseEntityWithTimestamps {
 	@Property()
+	@Index({ options: { expireAfterSeconds: 7776000 } })
```

Comment:
Can you also add a const on top with i do not know, something like SECONDS_OF_90_DAYS = 90 * 24 * 60 *60 and use it with expireAfterSeconds: SECONDS_OF_90_DAYS

Reply:
done